src/Core/Content/MeasurementSystem/Unit/ConvertedUnit.php

```diff
@@ -0,0 +1,16 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Content\MeasurementSystem\Unit;
+
+use Shopware\Core\Framework\Log\Package;
+
+/**
+ * @internal
```

Comment:
this shouldn't be internal or? If i understood correctly this is the DTO that is put into the product entity right? so it will be part of the HTTP API 

also the `unitSet` DTO is public, I think this should be public as well

Reply:
Yes it's mistakenly typed as internal 🙈